content/docs/technical_practices/azure/azure_make_vm_with_cli.md

```diff
@@ -0,0 +1,74 @@
+---
+# Page title as it appears in the navigation menu
+title: "Quickstart: Make a new VM using the command line"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 1
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Azure via CLI basic quickstart
```

Comment:
I don't think we want to duplicate or compete with Azure documentation, [for example](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-cli).

We are not going to be able to keep this as current or correct as Microsoft.